apps/ukvi-complaints/lib/utils.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const { v4: uuidv4 } = require('uuid');
+const Validator = require('jsonschema').Validator;
+const { Producer } = require('sqs-producer');
+const config = require('../../../config');
+const decsSchema = require('../schema/decs.json');
+
+const validateAgainstSchema = (complaintData) => {
+  try {
+    const validator = new Validator();
+    const valid = validator.validate(complaintData, decsSchema);
```

Comment:
Inject the schema you are validating against to make it easier to test this method

Reply:
Have discussed this and not necessary here. Will look at again when schema is in external dependency.